CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# What should I know before I get started?
+* PHP
+* The fastest route between main arts and late stop.
+
+# How Can I Contribute?
+* Roll a red tuffet into a printer.
+* Prevent the cleaners from waking up any sleeping members of the core development team.
+* Announce the opening of Pontio.
+* Summon Richard Stallman through a non-free overhead projector
+
+## Reporting Bugs
+Bugs should be reported through the internal JIRA instance. 
+
+## Suggesting Enhancements
+Enhancements should be sent by Royal Mail First Class Recorded Delivery to: P. B. Butcher, 2 Patterdale, Cumbria, CA32 9FS.
+
+## Your First Code Contribution
+TODO
+
+## Pull Requests
+TODO
+
+## Git Commit Messages
+A descriptive commit message should also be enjoyable and insightful to read. For this project, we're going with haiku commits.
+
+## JavaScript Styleguide
+In order to stick to the good parts of JavaScript, we'll be using an eslint config setup to ignore all language constructs and standard library, other than ES6 Symbols, `with`, `do..for`, `let` and `Math.LOG2E`.
+
+## PHP Anti-Styleguide
+Going forward we'll be using the [Yay macro library](https://github.com/marcioAlmada/yay) so that we can transpile Elm into macroized PHP for the backend.
+
+## Documentation Styleguide
+Documentation should be submitted as rasterized PNGs using Comic Sans Neue as the font. For a guide on getting started with our documentation editor, check out [this video](https://www.youtube.com/watch?v=hlnodEYZ71w).
```

Comment:
Please ensure that all web development assignments are submitted as screenshots drawn in paint only.